tools/torch/main.lua

```diff
@@ -181,8 +181,8 @@ if opt.subtractMean ~= 'none' then
 end
 
 local classes
-local confusion
-local validation_confusion
+local trainConfusion
+local validConfusion
```

Comment:
I liked `valid` only because it aligns well in the code with `train` (same length). But we could use `val` and in filenames **val.txt** and **val_db**.